diff --git a/capnp/web-session.capnp b/capnp/web-session.capnp
index f7cd457..694146a 100644
--- a/capnp/web-session.capnp
+++ b/capnp/web-session.capnp
@@ -470,7 +470,7 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   }
 
   interface WebSocketStream {
-    sendBytes @0 (message :Data) -> stream;
+    sendBytes @0 (message :Data $Go.name("msg")) -> stream;
     # Send some bytes.  WARNING:  At present, we just send the raw bytes of the WebSocket protocol.
     # In the future, this will be replaced with a `sendMessage()` method that sends one WebSocket
     # datagram at a time.
